protocol/messenger_communities.go

```diff
@@ -2305,21 +2305,44 @@ func (m *Messenger) ShareCommunity(request *requests.ShareCommunity) (*Messenger
 	if err := request.Validate(); err != nil {
 		return nil, err
 	}
+
+	var messageStatusLinkPreview common.StatusLinkPreview
 	response := &MessengerResponse{}
 
 	community, err := m.communitiesManager.GetByID(request.CommunityID)
 	if err != nil {
 		return nil, err
 	}
 
+	communityURL, err := m.ShareCommunityURLWithChatKey(community.ID())
+	if err != nil {
+		return nil, err
+	}
+
+	communityLinkPreview := &common.StatusCommunityLinkPreview{
```

Comment:
This is interesting.

I like the simplicity of this solution. We simply fill `StatusCommunityLinkPreview` inplace.
But this duplicates the code from `StatusUnfurler`:
https://github.com/status-im/status-go/blob/1794b93c16b1f68cdb7125b4b66cc8926da9eed2/protocol/linkpreview_unfurler_status.go#L115-L121

I thought that we will call `UnfurlURLs` here. But that would be definitely an overkill here.

Maybe we could extract a smth like `func BuildStatusCommunityLinkPreview(community *communities.Community)`? This will make the solution both simple and non-duplicated.